src/overlays/actors/ovl_En_Fish2/z_en_fish2.c

```diff
@@ -1091,15 +1091,15 @@ void func_80B2ADB0(EnFish2* this, Vec3f* vec, s16 arg2) {
 
     for (i = 0; i < ARRAY_COUNT(this->unk_3F8); i++, ptr++) {
         if (!ptr->unk_00) {
-            TexturePtr phi_v0;
+            TexturePtr texture;
 
             if (Rand_ZeroOne() < 0.5f) {
-                phi_v0 = gEffBubble2Tex;
+                texture = gEffBubble2Tex;
             } else {
-                phi_v0 = gEffBubble1Tex;
+                texture = gEffBubble1Tex;
             }
 
-            ptr->unk_20 = VIRTUAL_TO_PHYSICAL(SEGMENTED_TO_VIRTUAL(phi_v0));
+            ptr->unk_20 = (void*)VIRTUAL_TO_PHYSICAL(SEGMENTED_TO_VIRTUAL(texture));
```

Comment:
```suggestion
            ptr->unk_20 = (TexturePtr)VIRTUAL_TO_PHYSICAL(SEGMENTED_TO_VIRTUAL(texture));
```
Based on this feel like `unk_20` should actually be a `TexturePtr`.